apps/website-example/pages/_app.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Container,
+  createTheme,
+  css,
+  CssBaseline,
+  styled,
+  Theme,
+  ThemeOptions,
+  ThemeProvider
+} from '@mui/material'
+import {theme} from '@wepublish/ui'
+import {ApiV1, FooterContainer, WebsiteBuilderProvider, WebsiteProvider} from '@wepublish/website'
+import {setDefaultOptions} from 'date-fns'
+import {de} from 'date-fns/locale'
+import {AppProps} from 'next/app'
+import getConfig from 'next/config'
+import Head from 'next/head'
+import Script from 'next/script'
+import {PartialDeep} from 'type-fest'
+import {authLink} from '../src/auth-link'
+import {ReactComponent as Logo} from '../src/logo.svg'
+import {NextWepublishLink} from '../src/next-wepublish-link'
+import {SessionProvider} from '../src/session.provider'
+
+setDefaultOptions({
+  locale: de
+})
+
+const websiteExampleTheme = createTheme(theme, {} as PartialDeep<Theme> | ThemeOptions)
+
+const Spacer = styled('div')`
```

Comment:
Allows us to have the footer always at the bottom of the page, even if the main content is empty